build.yml

```diff
@@ -42,7 +42,7 @@ notebook:
 #      - source: Examples/SurrMod/RIPE
 #        match: "RIPE_.*"
       - source: Examples/Tools
-      - source: Examples/UnitModels
+      - source: Examples/UnitModels/Operations
```

Comment:
Has this been updated in `build-ci.yml` as well? If not, it should.

Reply:
It has not. I would recommend we update the developer guide https://github.com/IDAES/examples-pse/blob/main/README-developer.md to make updating both build files more clear.